src/dos/cdrom_image.cpp

```diff
@@ -534,7 +534,7 @@ bool CDROM_Interface_Image::LoadCueSheet(char *cuefile)
 	}
 	df->Close();
 	delete df;
-	istringstream inString = istringstream(dosfilebuf);
+	istringstream inString; //  = istringstream(dosfilebuf);
```

Comment:
This is probably bad. What's this even doing?

Reply:
This needs to be `istringstream inString(dosfilebuf);`.

Reply:
I'll get this fixed today